apps/omg_watcher/test/integration/in_flight_test.exs

```diff
@@ -0,0 +1,100 @@
+# Copyright 2018 OmiseGO Pte Ltd
```

Comment:
this file duplicates the integration tests already under `inflight_exit_test.exs`, please merge.

Also I'd envision this as a one joint integration test, which would poke all the integrations - including that one, being checking if the child chain correctly spent the utxos.

Not 100% sure, but for now this is my vote, as otherwise the integration tests will get uncomfortably long to run.

CC @paulperegud